velox/common/file/FileTest.cpp

```diff
@@ -56,10 +56,10 @@ void readData(ReadFile* readFile) {
   char tail[7];
   std::vector<folly::Range<char*>> buffers = {
       folly::Range<char*>(head, sizeof(head)),
-      folly::Range<char*>(nullptr, 500000),
+      folly::Range<char*>(ReadFile::skipMarker(), 500000),
```

Comment:
You should make similar changes inside `S3FileSystemTest.cpp`.

Reply:
Thanks I'll make the changes.